appveyor.yml

```diff
@@ -39,7 +39,7 @@ install:
 platform: x64
 
 # build Configuration, i.e. Debug, Release, etc.
-configuration: Release - LLVM
+configuration: Release LLVM
```

Comment:
im not sure this should've been changed, since everywhere else it's still using `Release - LLVM` and this change is probably what's causing appveyor build to fail